cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -25,6 +30,39 @@
 log = logging.getLogger('edx.modulestore')
 
 
+def fetch_from_about(modulestore, course_id, attribute):
+    """ Get about attribute from modulestore """
+    usage_key = course_id.make_usage_key('about', attribute)
+    try:
+        value = modulestore.get_item(usage_key).data
+    except ItemNotFoundError:
+        value = None
+    return value
+
+
+def fetch_course_information_detail(modulestore, course, about_information):
+    """
+    Fetch the course information attribute for the given course's attribute
+    from the correct persistence location and return its value.
+    """
+    if about_information[2]:
+        return fetch_from_about(modulestore, course.id, about_information[0])
```

Comment:
Likewise with this array index reference -- will it always be the first item in the list?